assignments/arrays.js

```diff
@@ -71,20 +71,32 @@ console.log(`Car 33 is a ` + inventory[32].car_year +" " + inventory[32].car_mak
 let lastCar = 0;
 lastCar = inventory;
 for (i = 0; i < lastCar.length-1; i++ ){
-   //console.log(inventory.car_make);
-
+   //
 }
 console.log(`Make:`+ " " + inventory[i].car_make + " "+ `Model:` + " " + inventory[i].car_model);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+for (let i = 0; i < inventory.length; i++){
+    let model = inventory[i].car_model;
+    carModels.push(model);
+    carModels.sort();
+    
+    }
+
+    console.log(carModels);
+
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+ for(i = 0; i < inventory.length; i++){
+     const year = inventory[i].car_year;
```

Comment:
this definitely works. a shorter way of doing this is placing the `inventory[i].car_year` directly into the push of carYears like so
`carYears.push(inventory[i].car_year);`